extensions/amp-gwd-animation/0.1/amp-gwd-animation.css

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Class used to disable GWD animations.
+ */
+.i-amphtml-gwd-animation-disabled [class*=gwdanimation] {
```

Comment:
Just to confirm: this is your normal practice as well, right?

Reply:
Which part are you referring to? Our regular non-AMP runtime does not have need to disable animations globally, so we don't employ any CSS quite like this.

Reply:
I see. Did you try `.i-amphtml-gwd-animation-disabled * {animation:none}` and found it less performant?

Reply:
Leaving as-is for now as discussed.